openage/convert/service/export/png/binpack.pxd

```diff
@@ -7,9 +7,11 @@ cdef class Packer:
     cdef dict mapping
 
     cdef void pack(self, list blocks)
```

Comment:
If you do the changes I suggested and pass a `vector` instead of a list in this function, you can then also pass it by reference:

```suggestion
    cdef void pack(self, vector[block_t] &blocks)
```

This will avoid a copy of the vector and thus save some time.

Reply:
see https://github.com/SFTtech/openage/pull/1671#discussion_r1864312773